oaknational/Oak-Web-Application
# PUPIL-415 replace typography related components with oak-components

## Description

Replace typography related components with relevant `oak-components`.
Most of the components has beed updated. There should be no visual changes.

I've documented what is remaining/needs to be looked at here (at the bottom of document):
https://www.notion.so/oaknationalacademy/Replacing-SharedComponents-in-OWA-app-with-oak-components-de39c55b33e947bbbd30888547f5b10e

## Issue(s)

Fixes #PUPIL-415

## How to test

1. Go to https://deploy-preview-2228--oak-web-application.netlify.thenational.academy
2. Check the main journeys/menus/etc
3. There should be no visual changes
